week1/homework/src/index.js

```diff
@@ -1,3 +1,53 @@
-'use strict';
+/* eslint-disable no-unused-vars */
 
-// Write the homework code in this file
+"use strict";
+
+const state = 10;
+
+const http = require("http");
+
+const url = "http://localhost:8080";
+
+function handleRequest(request, response) {
+    response.setHeader("Content-Type", "text/html");
+    switch (url + request.url) {
+        case "http://localhost:8080/":
+            response.write(`<em><h1>This is the main page</h1></em>`);
+            break;
+        case stateUrl:
+            response.write(`<h1>This is the original state: ${state}</h1>`);
+            break;
+        case addUrl:
+            let add = state + 1;
```

Comment:
You are not modifying the state. If I call `/add` twice after starting the program I expect `state` to be 12. In your code `state` is _always_ 10.